### DISCUSSION TITLE
How can I check what version is optimally compatible with my RN version?

### DISCUSSION THREAD

**Discussion by ablack34** [reduxjs/redux-toolkit#3141] [Q&A]
Hello,

I'm new to the RN ecosystem and I am trying to update dependencies to compatible versions with my RN version. Is there a simple way to check what versions of this library are compatible with certain versions of RN? 

Thanks! :)

**Comment by markerikson** [comment]
Redux Toolkit by itself actually has _nothing_ to do with React.  So, it does not matter what version of React or RN you're using.

The only part of RTK that relates to React at all is if you're using RTK Query's React hooks, in which case you need at least React 16.8 or greater (which came out four years ago).
